src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,17 @@
 
 public class MainApp {
 
+  Machine excavator = new Excavator();
+  Machine bulldozer = new Bulldozer();
+  Machine truck = new Truck();
+  Machine[] machines = new Machine[] {excavator, bulldozer, truck};
+  public void getMachineCond() {
```

Comment:
The method name 'getMachineCond' is not informative enough and does not describe what the method actually does. Consider renaming it to a more descriptive name, such as 'operateMachines', which indicates that the machines are being operated (started and stopped).